libr/cons/dietline.c

```diff
@@ -176,7 +179,9 @@ static int r_line_readchar_win(int *vch) { // this function handle the input in
 	GetConsoleMode (h, &mode);
 	SetConsoleMode (h, 0);	// RAW
 	*vch = 0;
+	void *bed = r_cons_sleep_begin ();
```

Comment:
Windows doesn't like that (already defined in the if).
Declare it above and define it after.

Reply:
Yes, you are right. I haven't tested that on Windows yet.

Reply:
I just read the AppVeyor output :p